scout/server/blueprints/alignviewers/templates/alignviewers/igv_sashimi_viewer.html

```diff
@@ -77,8 +77,8 @@ <h3>{{case}}: Splice junctions</h3>
                                   colorBy: 'strand',
                                   thicknessBasedOn: 'numUniqueRead',
                                   labelUniqueReadCount: true,
-                                  url: "{{ url_for('alignviewers.unindexed_remote_static', file=track.splicej_bed) }}",
-                                  indexURL: "{{ url_for('alignviewers.unindexed_remote_static', file=track.splicej_bed_index) }}",
+                                  url: "{{ url_for('alignviewers.remote_static', file=track.splicej_bed) }}",
+                                  indexURL: "{{ url_for('alignviewers.remote_static', file=track.splicej_bed_index) }}",
```

Comment:
Didn't we already? 👀

Reply:
Not for this track, last time I've modified the bigwig one